api/graphql/schema.graphql

```diff
@@ -240,6 +240,31 @@ enum PoolStatus {
     NOT_TAKING_APPLICATIONS
 }
 
+type PoolAdvertisement {
+    id: ID!
+    expiryDate: String @rename(attribute: "expiry_date")
```

Comment:
Can we get a zoned expiry date back here?  Or at least in UTC?

Reply:
:tada: https://lighthouse-php.com/master/api-reference/scalars.html#datetimetz

Reply:
Thanks!